### ISSUE TITLE
Can't run SDL2 version due to missing glyph in font.

### ISSUE THREAD

**Issue by ShimmerFairy** [angband/angband#5079]
Trying to run the SDL2 version of Angband 4.2.2 (what gentoo currently offers) only gives up this unhelpful error message:

```
$ angband -msdl2
angband: cant render surface for cache in font '8x13x.fon': Couldn't find glyph
```

The file in question does exist, so it's not that the font itself can't be found. I also tried installing Angband 4.2.3 to /usr/local, and I got the same error.

**Comment by backwardsEric** [comment]
Thank you for the report.  Some more information would be helpful since spinning up an instance of Gentoo to test this isn't trivial and I can't reproduce the failure running the SDL2 front end on Debian Buster.  In particular:

- What's the CPU architecture?  x86_64, x86, or something else?
- Which compiler (name and version) was used to build Angband?  If a different compiler was used to build sdl2, what was that compiler?
- Which version of sdl2 was used?  What's the version for sdl2-ttf?
- How did you configure Angband when compiling it?

**Comment by ShimmerFairy** [comment]
I have an x86_64 computer, and both Angband and sdl2 would've been built with gcc 11.2.0. I'm using SDL2 version 2.0.16, with sdl2-ttf version 2.0.15. When building the latest version myself the only option I gave `./configure` was the one needed to enabled SDL2 support. As for the gentoo version, I have the X, ncurses, and sdl USE flags enabled (and sound disabled), which enables the necessary configure options.

**Comment by backwardsEric** [comment]
This appears to be due to a change in FreeType 2.11 that has since been restored to behave as it did with FreeType 2.10.4 and before.  https://gitlab.freedesktop.org/freetype/freetype/-/issues/1076 has a description of the issue.  Using FreeType 2.10.4 or a patched version of 2.11 that includes the changes mentioned in that issue should resolve this.

**Comment by NickMcConnell** [comment]
I'm assuming this can be closed now - let me know if I need to reopen it.
